design/proposals/encoded_token_response.md

```diff
@@ -0,0 +1,165 @@
+# Encoded Token Response - Solution Design
+
+## Table of Contents
+  * [Table of Contents](#table-of-contents)
+  * [Glossary](#glossary)
+  * [Useful links](#useful-links)
+  * [Background](#background)
+  * [Issue description](#issue-description)
+  * [Solution - Accept HTTP Header](#solution---accept-http-header)
+    + [Design](#design)
+    + [Backwards compatibility](#backwards-compatibility)
+  * [Solution - URL Patten](#solution---url-patten)
+    + [Design](#design-1)
+    + [Backwards compatibility](#backwards-compatibility-1)
+  * [Preferred Solution](#preferred-solution)
+  * [Security](#security)
+  * [Test Plan](#test-plan)
+  * [Documentation](#documentation)
+  * [Open questions](#open-questions)
+
+## Glossary
+
+| **Term** | **Description**  |
+|----------|------------------|
+| encoded  |base64 encoded    |
+| JWS      |json web signature|
+| JWT      |json web token    |
+
+
+## Useful links
+1. [HTTP Content Negotiation Best Practices](https://restfulapi.net/content-negotiation/)
+2. MDN web docs:
+    1. [HTTP Accept Header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept)
+    2. [HTTP Accept-Encoding Header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Encoding)
+    3. [HTTP Content-Type Header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type)
+    4. [HTTP Content-Encoding Header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Encoding)
+3. [Rails Routing Guides](https://guides.rubyonrails.org/routing.html#route-globbing-and-wildcard-segments)
+4. [Correct Usage of JSON](https://stackoverflow.com/questions/477816/what-is-the-correct-json-content-type)
+
+## Background
+
+As a Conjur User or Host, I want to send authentication request to `authn` authenticator (for example).
+So that I receive a JWT access token.
```

Comment:
i would emphasize that you want a "ready-to-use" base64 encoded Conjur access token instead of needing to encode it yourself

Reply:
and i think that the "As a" language is abit hard to understand here. I would state the current behaviour (currently, when a customer sends an authn request...), it's problem and the desired behaviour

Reply:
i now see that you already wrote it later. in that case i think the whole sentence is redundant

Reply:
Changed